### PR TITLE
Remove iOS 15 support

### PR DESCRIPTION
**Pull request by dfeinzimer**
There are two deprecation warnings that I'll leave to be fixed separately because I expect them to be a bit more involved.

### CODE REVIEW

File: Sources/ArcGISToolkit/Utility/AsyncImageView.swift [hunk 1]
```diff
@@ -94,8 +94,7 @@ public struct AsyncImageView: View {
                 }
                 .padding([.top, .bottom])
             }
-            if #available(iOS 16.0, macCatalyst 16.0, *),
-               let progressInterval = viewModel.progressInterval {
+            if let progressInterval = viewModel.progressInterval {
 #if canImport(Charts)
```
Comment (mhdostal) [inline Sources/ArcGISToolkit/Utility/AsyncImageView.swift:98]: You can remove all instances of `#if canImport(Charts)` from the code. `Charts` is only available on iOS 16; now that it's the minimum, everyone gets Charts!

**Review by mhdostal — CHANGES_REQUESTED** [review]

File: AuthenticationExample/AuthenticationExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -239,7 +239,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```
Comment (CalebRas) [inline AuthenticationExample/AuthenticationExample.xcodeproj/project.pbxproj:242]: Don't forget to update the repo README [here](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit?tab=readme-ov-file#requirements) too!

File: Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift [hunk 3]
```diff
@@ -149,7 +108,7 @@ private extension TextInput {
                     }
                 }
             }
-            .scrollContentBackgroundHidden()
+            .scrollContentBackground(.hidden)
```
Comment (CalebRas) [inline Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift:111]: Does [this property](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/main/Sources/ArcGISToolkit/Components/Bookmarks/Bookmarks.swift#L176-L180) need updated too?

**Review by CalebRas — COMMENTED** [review]
Looks good overall!

File: AuthenticationExample/AuthenticationExample.xcodeproj/project.pbxproj (hunk 2 above)
Comment (dfeinzimer) [reply AuthenticationExample/AuthenticationExample.xcodeproj/project.pbxproj:242]: Got it, thanks!

File: Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift (hunk 3 above)
Comment (dfeinzimer) [reply Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift:111]: Good question, this area has is being updated in #549.

File: Examples/Examples/FeatureFormExampleView.swift [hunk 4]
```diff
@@ -84,19 +84,13 @@ struct FeatureFormExampleView: View {
...
+                        ToolbarItem(placement: .navigationBarLeading) {
                             Button("Cancel", role: .cancel) {
                                 cancelConfirmationIsPresented = true
                             }
                         }
-                    }
-                    
-                    ToolbarItem(placement: .navigationBarTrailing) {
-                        if model.isFormPresented {
+                        ToolbarItem(placement: .navigationBarTrailing) {
```
[the remark this answers is not in the document]
Comment (dfeinzimer) [reply Examples/Examples/FeatureFormExampleView.swift:93]: I would like to but I don't think there's a way to use `ToolbarItemGroup` and get the buttons to appear on either side of the navigation title.

**Review by CalebRas — APPROVED** [review]

**Review by mhdostal — APPROVED** [review]

**Pull request merged**